extension/public/static/manifest.json

```diff
@@ -1,8 +1,9 @@
 {
   "name": "Lyra",
-  "version": "1.0",
-  "description": "Build an Extension!",
-  "permissions": ["activeTab", "declarativeContent", "storage"],
```

Comment:
`activeTab` and `declarativeContent` permissions aren't actually needed over anymore since i refactored the messaging system. 

The Chrome store makes you justify any permissions you use when you submit, so removing!